web/src/message_list_tooltips.ts

```diff
@@ -276,7 +289,8 @@ export function initialize() {
     message_list_tooltip(
         [".rendered_markdown time", ".rendered_markdown .copy_codeblock"].join(","),
         {
-            content: timerender.get_markdown_time_tooltip,
+            // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+            content: timerender.get_markdown_time_tooltip as tippy.Content,
```

Comment:
This may be annoying to avoid the cast; it's expecting a function with type `((ref: Element) => string | Element | DocumentFragment);`, and our function has type `HTMLElement => string | DocumentFragment`.

Reply:
As you stated `timerender.get_markdown_time_tooltip` is of type `HTMLElement => string | DocumentFragment` but it is expecting `content` parameter of type `tippy.Content` (an optional field of upstream `tippy.Props` ).

Reply:
Yeah; that Tippy type is a union, so it could work without the cast if not for the `Element`/`HTMLElement` distinction, I think.

Reply:
Yes, I agree!